jwst/pipeline/calwebb_image2.py

```diff
@@ -51,8 +50,12 @@ def process(self, input):
 
             # Save result
             suffix = 'cal'
+            if isinstance(input, datamodels.CubeModel):
```

Comment:
Yes, there is a reg test for ints (test_image2pipeline_2b). That's how I caught it. It was the 1 failing test for the past couple days.